src/app/user/login/login.component.html

```diff
@@ -1,2 +1,58 @@
+<!-- 
 <div class="login-main-container">
+
+
+</div>
+-->
+<div class="main-container-login" id="main-login-container">
+    <div class="login-container">
+        <div class="login-inputs-container">
+            <!-- Container for sign in title-->
+            <div class="login-title">
+                <h2>Sign in to AltruSight </h2>
+            </div>
+
+            <!-- Container for login inputs-->
+            <div class="login-inputs">
+                <!-- Email input -->
+                <mat-form-field appearance="outline">
+                    <mat-label>Email</mat-label>
+                    <input matInput placeholder="Enter Your Email">
+                    <mat-icon matSuffix>email</mat-icon>
+                </mat-form-field>
+    
+                <!-- Password input -->
+                <mat-form-field appearance="outline">
+                    <mat-label>Password</mat-label>
+                    <input matInput [type]="hide ? 'password' : 'text'">
+                    <button mat-icon-button matSuffix (click)="hide = !hide" [attr.aria-label]="'Hide password'" [attr.aria-pressed]="hide">
+                        <mat-icon>{{hide ? 'visibility_off' : 'visibility'}}</mat-icon>
+                    </button>
+                    <mat-hint><a class="forgot-password" href="login">Forgot Password</a></mat-hint>
```

Comment:
Instead of changing the color, I changed it to say "Forgot password? Click here!". I thought adding color would take away from the clean look so I think this change makes it clear that they can change their password through that link.